src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -15,10 +15,12 @@ type Props = {
   className?: string;
   /**
    * How much space there should be between the buttons.
+   * 'max' does not work with 'vertical' orientation.
```

Comment:
When thinking about the documentation for this component, I realized we're not surfacing this unexpected behavior anywhere.